live/lib/auth_app/accounts/user.ex

```diff
@@ -8,38 +8,29 @@ defmodule AuthApp.Accounts.User do
     field :hashed_password, :string, redact: true
     field :current_password, :string, virtual: true, redact: true
     field :confirmed_at, :utc_datetime
+    field :authenticated_at, :utc_datetime, virtual: true
 
     timestamps(type: :utc_datetime)
   end
 
   @doc """
-  A user changeset for registration.
+  A user changeset for registering or changing the email.
 
-  It is important to validate the length of both email and password.
-  Otherwise databases may truncate the email without warnings, which
-  could lead to unpredictable or insecure behaviour. Long passwords may
-  also be very expensive to hash for certain algorithms.
+  It requires the email to change otherwise an error is added.
 
   ## Options
 
-    * `:hash_password` - Hashes the password so it can be stored securely
-      in the database and ensures the password field is cleared to prevent
-      leaks in the logs. If password hashing is not needed and clearing the
-      password field is not desired (like when using this changeset for
-      validations on a LiveView form), this option can be set to `false`.
-      Defaults to `true`.
-
     * `:validate_email` - Validates the uniqueness of the email, in case
       you don't want to validate the uniqueness of the email (like when
       using this changeset for validations on a LiveView form before
       submitting the form), this option can be set to `false`.
       Defaults to `true`.
```

Comment:
Not your fault but this description is really bad :D

```suggestion
    * `:validate_email` - Set to false if you don't want to validate the
      uniqueness of the email. Defaults to `true`.
```